site/admin_server.md

```diff
@@ -0,0 +1,58 @@
+## Admin Server Configuration
+
+The Create Domain Tool allows you to configure the admin server using a domain model. These examples show how some common configurations can be represented in the model.
+
+### Using the Default Admin Server Configuration
+
+When the Create Domain Tool is run, the templates associated with your domain type will automatically create an admin server named **AdminServer**, with default values for all the attributes. If you don't need to change any of these attributes, such as **ListenAddress** or **ListenPort**, or any of the sub-folders, such as **SSL** or **ServerStart**, nothing needs to be added to the model.
```

Comment:
Use back tick to surround code words like AdminServer and ListenAddress instead of asterisks.

Reply:
Changed these